src/webhook/mutation/pod_mutator/oneagent_mutation/config.go

```diff
@@ -9,12 +9,14 @@ var (
 )
 
 const (
-	preloadEnv           = "LD_PRELOAD"
-	networkZoneEnv       = "DT_NETWORK_ZONE"
-	proxyEnv             = "DT_PROXY"
-	dynatraceMetadataEnv = "DT_DEPLOYMENT_METADATA"
-	releaseVersionEnv    = "DT_RELEASE_VERSION"
-	releaseProductEnv    = "DT_RELEASE_PRODUCT"
+	preloadEnv             = "LD_PRELOAD"
+	networkZoneEnv         = "DT_NETWORK_ZONE"
+	proxyEnv               = "DT_PROXY"
+	dynatraceMetadataEnv   = "DT_DEPLOYMENT_METADATA"
+	releaseVersionEnv      = "DT_RELEASE_VERSION"
```

Comment:
thats the most nitpicky thing, but I would prefer a newline here for better visbility
```suggestion
	dynatraceMetadataEnv   = "DT_DEPLOYMENT_METADATA"
	
	releaseVersionEnv      = "DT_RELEASE_VERSION"
```